app/components/register.jsx

```diff
@@ -106,7 +106,7 @@ export class Register extends Component {
                   </h2>
                   <div className="form-group">
                     <label htmlFor="dex_title">Title</label>
```

Comment:
ya, but in that case i think it's weird because with Living Dex - that'd be something completely reasonable to name your dex, while ashketchum10 isn't (except for that one user lols). because of that, i think it'd be more confusing than anything to put it there. and since - like you mentioned - it doesn't provide much benefit on the register page other than looking cool, i'd rather not introduce a possible point of confusion without much utility.